14/umbraco-cms/reference/content-delivery-api/README.md

```diff
@@ -14,9 +14,19 @@ The Delivery API is an opt-in feature in Umbraco. It must be explicitly enabled
 
 ### Enable the Content Delivery API
 
+You can enable the Delivery API when creating you project initially by using the `--use-delivery-api` or `-da` flag. This will automatically add the necessary configuration to your project.
```

Comment:
```suggestion
When creating your project, you can enable the Delivery API using the `--use-delivery-api` or `-da` flag. This will automatically add the necessary configuration to your project.
```